packages/license-collector/README.md

```diff
@@ -0,0 +1,31 @@
+# Wire
+
+This repository is part of the source code of Wire. You can find more information at [wire.com](https://wire.com) or by contacting opensource@wire.com.
+
+You can find the published source code at [github.com/wireapp](https://github.com/wireapp).
+
+For licensing information, see the attached LICENSE file and the list of third-party licenses at [wire.com/legal/licenses/](https://wire.com/legal/licenses/).
+
+## LicenseCollector
+
+Wire's internal license collection tool.
```

Comment:
What does `Wire's internal license collection tool` do?

Reply:
It collects all dependencies of given projects and retrieves their licenses.